docs/docs/guide/project/dependencies.md

```diff
@@ -194,3 +194,11 @@ pod install
 
 > [!WARNING] 
 > CocoaPods dependencies are not compatible with workflows like `build` or `test` that run `xcodebuild` right after generating the project. They are also incompatible with binary caching and selective testing since the fingerprinting logic doesn't account for the Pods dependencies.
+
+## Objective-C Dependencies
+
+When integrating Objective-C dependencies, the inclusion of certain flags on the consuming target may be necessary to avoid runtime crashes as detailed in [Apple Technical Q&A QA1490](https://developer.apple.com/library/archive/qa/qa1490/_index.html).
+
+Since the build system and Tuist have no way of inferring whether the flag is necessary or not, and since the flag comes with potentially undesirable side effects, Tuist will not automatically apply any of these flags, and because Swift Package Manager considers `-ObjC` to be included via an `.unsafeFlag` most packages cannot include it as part of their default linking settings when required.
+
+Consumers of Objective-C dependencies (or internal Objective-C targets) should apply  `-ObjC` or `-force_load` flags when required by setting `OTHER_LDFLAGS` on consuming targets.
```

Comment:
The `app_with_spm_dependencies` fixture requires the `-ObjC` flag. You can double check that by running the app on the simulator. Let's add it and we can link to it as an example through references, once a README is added: https://docs.tuist.io/reference/examples/app_with_build_rules.html

Reply:
@thedavidharris once this is resolved, we can merge this 🙂

Reply:
Added!

Reply:
Thanks! However, the documentation here still misses a link to the fixture 🙏